src/Game.js

```diff
@@ -0,0 +1,101 @@
+const MissionUtils = require('@woowacourse/mission-utils');
+const gameControlValidation = require('./validation/gameControlValidation.js');
+const gameInputValidation = require('./validation/gameInputValidation.js');
+
+const MESSAGE = {
+  WIN: '3스트라이크\n3개의 숫자를 모두 맞히셨습니다! 게임 종료',
+  START: '숫자 야구 게임을 시작합니다.',
+  INPUT: '숫자를 입력해 주세요 : ',
+  RESTART_INPUT: '게임을 새로 시작하려면 1, 종료하려면 2를 입력하세요. ',
+};
+const RESTART = 1;
+const END = 2;
+
+module.exports = class Game {
+  constructor() {
+    this.computerNumbers;
+    MissionUtils.Console.print(MESSAGE.START);
+    this.gameInit();
+  }
+
+  gameInit() {
+    this.computerNumbers = [...this.getRandomNumbers()].join('');
+    this.getUserNumberInput();
+  }
+
+  getRandomNumbers() {
+    const computerNumbers = new Set();
+    while (computerNumbers.size < 3) {
+      const number = MissionUtils.Random.pickNumberInRange(1, 9);
+      if (!computerNumbers.has(number)) computerNumbers.add(number);
+    }
+    return computerNumbers;
```

Comment:
set을 사용하셔서 그런지 로직이 단순하고 좋네요!:) set은 이미 있는 원소는 추가해도 사이즈가 늘지 않기때문에, 이렇게도 가능할것 같아요!:) 
```suggestion
  getRandomNumbers() {
    const computerNumbers = new Set();
    while (computerNumbers.size < 3) {
      const number = MissionUtils.Random.pickNumberInRange(1, 9);
      computerNumbers.add(number);
    }
    return computerNumbers;
```

Reply:
저도 다른분거 리뷰하면서 set이란 자료구조를 알게되면서 그런 부분에 놀랐어요! ㅋㅋㅋㅋ Set쓰신것 부터가 👍🏻멋지세요!:) 화이팅입니다!!